Corzbank.Web/ClientApp/src/app/header/header.component.ts

```diff
@@ -22,4 +28,51 @@ export class HeaderComponent implements OnInit {
       console.log('The dialog was closed');
     });
   }
+
+  get isAuthenticated() {
+    let isLogedin = Number(this.storageService.getItem(StorageTypeEnum.localStorage, "isLogedin"));
+    if (isLogedin == 1) {
+      return true;
+    }
+    return false;
+  }
+
+  isUserAuthenticated() {
+    const accessToken = JSON.parse(this.storageService.getItem(StorageTypeEnum.localStorage, "accessToken") as string);
+
+    if (accessToken && !this.jwtHelper.isTokenExpired(accessToken)) {
+      return true;
+    }
+    else {
+      let isLogedin = Number(this.storageService.getItem(StorageTypeEnum.localStorage, "isLogedin"));
+
+      if (isLogedin == 1) {
+        let refreshToken: string;
+
+        this.authenticationService.refreshTokensForUser(accessToken).subscribe((response: any) => {
+
+          this.storageService.setItem(StorageTypeEnum.localStorage, "accessToken", JSON.stringify(response.accessToken));
+          refreshToken = response.refreshToken_;
+
+          if (refreshToken && !this.jwtHelper.isTokenExpired(refreshToken)) {
+            return true;
+          }
+          else {
+            this.storageService.clear(1);
```

Comment:
Use Enum Values instead of hardcoded numeric (1).

Reply:
resolved